test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetUpdateCommandTests.cs

```diff
@@ -1677,5 +1677,140 @@ public void UpdateCommand_Self_FailsWithMoreThanOneSource()
                 result.AllOutput.Contains(NuGetResources.Error_UpdateSelf_Source);
             }
         }
+
+        [Fact]
+        public async Task UpdateCommand_NF_Project_Success()
+        {
+            using (var packagesSourceDirectory = TestDirectory.Create())
+            using (var solutionDirectory = TestDirectory.Create())
+            using (var workingPath = TestDirectory.Create())
+            {
+                // setup directories
+                var projectDirectory1 = Path.Combine(solutionDirectory, "proj1");
+                var projectDirectory2 = Path.Combine(solutionDirectory, "proj2");
+                var packagesDirectory = Path.Combine(solutionDirectory, "packages");
+
+                // create packages IDs
+                var a1 = new PackageIdentity("A", new NuGetVersion("1.0.0"));
+                var a2 = new PackageIdentity("A", new NuGetVersion("2.0.0"));
+
+                var b1 = new PackageIdentity("B", new NuGetVersion("1.0.0"));
+                var b2 = new PackageIdentity("B", new NuGetVersion("2.0.0"));
+
+                // create packages
+                var a1Package = Util.CreateNFTestPackage(
+                    a1.Id,
+                    a1.Version.ToString(),
+                    packagesSourceDirectory);
+
+                var a2Package = Util.CreateNFTestPackage(
```

Comment:
Are you making use of this `a2Package` and `b2Package`?

Reply:
On the code no, but the package file from that is used in the test update.